.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,26 @@
-^ Make sure to prefix the Title with [WIP]/[WIPTEST]
+<!-- Make sure to prefix the PR Title in the following flow...
 
-Purpose or Intent
-=================
+   [WIP] - Work in Progress
+   [WIPTEST] - Work in Progress with PRT Testing
+   [RFR] - Ready for 1st Review
+   [1LP][WIP] - 1st Level Pass, Work in Progress
+   [1LP][WIPTEST] - 1st Level Pass, Work in Progress with PRT Testing
+   [1LP][RFR] - 1st Level Pass, Ready for 2nd Review
 
-__Fixing__ X because it is currently doing Y which is incorrect. It should be doing Z.
+-->
 
-or
+## Purpose or Intent
```

Comment:
Can we please encourage people to add Screenshots/Gifs wherever make sense. Just add a line about that in here. Because pictures speak 1000 words as they say. ;)

Reply:
thanks for adding that :)